tests/unit/components/test_braveadblock.py

```diff
@@ -0,0 +1,320 @@
+        ResourceType.stylesheet,
+    ),
+    (
+        "https://www.ruv.is/sites/default/files/styles/2000x1125/public/fr_20180719_091367_1.jpg?itok=0zTNSKKS&timestamp=1561275315",
+        "https://www.ruv.is/frett/2020/04/23/today-is-the-first-day-of-summer",
+        ResourceType.image,
+    ),
+]
+
+NOT_OKAY_URLS = [
+    (
+        "https://pagead2.googlesyndication.com/pcs/activeview?xai=AKAOjsvBN5MuZsVQyE7HD18bD-JjK589TD3zkugwCoLE2C5nP26WFNCQb8WwxzZTelPEHwwnhaOCsGxYc8WeFgYZLReqLYl8r9BtAQ6r83OHa04&sig=Cg0ArKJSzKMgXuVbXAD1EAE&adk=1473563476&tt=-1&bs=1431%2C473&mtos=120250,120250,120250,120250,120250&tos=120250,0,0,0,0&p=60,352,150,1080&mcvt=120250&rs=0&ht=0&tfs=5491&tls=125682&mc=1&lte=0&bas=0&bac=0&if=1&met=ie&avms=nio&exg=1&md=2&btr=0&lm=2&rst=1587887205533&dlt=226&rpt=1849&isd=0&msd=0&ext&xdi=0&ps=1431%2C7860&ss=1440%2C810&pt=-1&bin=4&deb=1-0-0-1192-5-1191-1191-0-0-0&tvt=125678&is=728%2C90&iframe_loc=https%3A%2F%2Ftpc.googlesyndication.com%2Fsafeframe%2F1-0-37%2Fhtml%2Fcontainer.html&r=u&id=osdtos&vs=4&uc=1192&upc=1&tgt=DIV&cl=1&cec=1&wf=0&cac=1&cd=0x0&itpl=19&v=20200422",
+        "https://google.com",
+        ResourceType.image,
+    ),
+    (
+        "https://e.deployads.com/e/myanimelist.net",
+        "https://myanimelist.net",
+        ResourceType.xhr,
+    ),
+    (
+        "https://c.amazon-adsystem.com/aax2/apstag.js",
+        "https://www.reddit.com",
+        ResourceType.script,
+    ),
+    (
+        "https://c.aaxads.com/aax.js?pub=AAX763KC6&hst=www.reddit.com&ver=1.2",
+        "https://www.reddit.com",
+        ResourceType.script,
+    ),
+    (
+        "https://pixel.mathtag.com/sync/img/?mt_exid=10009&mt_exuid=&mm_bnc&mm_bct&UUID=c7b65ea6-76cc-4700-b0c7-6dbcd10820ed",
+        "https://damndelicious.net/2019/04/03/easy-slow-cooker-chili/",
+        ResourceType.image,
+    ),
+]
+
+
+def run_function_on_dataset(given_function):
+    """Run the given function on a bunch of urls.
+
+    In the data folder, we have a file called `adblock_dataset.tsv`, which
+    contains tuples of (url, source_url, type) in each line. We give these
+    to values to the given function, row by row.
+    """
+
+    def dataset_type_to_enum(type_int: int) -> ResourceType:
+        """Translate the dataset's encoding of a resource type to Qutebrowser's."""
+        if type_int == 0:
+            return ResourceType.unknown
+        elif type_int == 1:
+            return ResourceType.image
+        elif type_int == 2:
+            return ResourceType.stylesheet
+        elif type_int == 3:
+            return ResourceType.media
+        elif type_int == 4:
+            return ResourceType.script
+        elif type_int == 5:
+            return ResourceType.font_resource
+        elif type_int == 6:
+            return ResourceType.xhr
+        else:
+            assert type_int == 7
+            return ResourceType.sub_frame
+
+    dataset_path = os.path.join(THIS_DIR, "data", "adblock_dataset.tsv")
```

Comment:
Can you please gzip this file (and the other giant data files) similar to 9a6d027b11c40bb68445f0a68332e35db21cbf92?